src/ol_infrastructure/infrastructure/aws/network/Pulumi.infrastructure.aws.network.CI.yaml

```diff
@@ -7,6 +7,7 @@ config:
   data_vpc:cidr_block: 172.17.0.0/16
   k8s_vpc:cidr_block: 172.30.0.0/17
   k8s_vpc:k8s_service_subnet: 172.30.48.0/20
+  micromasters_vpc:cidr_block: 172.23.0.0/16
```

Comment:
This should get deleted since we are deprecating the VPC for Micromasters once we complete the migration.

Reply:
Yeah we should probably have a conversation about how we use Git as a team. I keep trying to git pull --rebase which keeps tripping over commits people are merging in to main. Does everyone fast forward or something?

I'm used to everyone working in branches, rebasing main onto them, and then squash rebase -> merge when done.